python-catkin-tools/install.bash

```diff
@@ -0,0 +1,6 @@
+#! /usr/bin/env bash
+
+tue-install-system-now python-catkin-tools
+
+# add optional psutil support
```

Comment:
What does this do?

Reply:
you added it the ros-catkin target, we refactored it. So can you tell us?

Reply:
Configuring Memory Use
In addition to CPU and load limits, catkin build can also limit the number of running jobs based on the available memory, using the hidden --mem-limit flag. This flag requires installing the Python psutil module and is useful on systems without swap partitions or other situations where memory use needs to be limited.

This is what it should do according to the docs

Reply:
Ah, I did add this., you moved the installation target but changed the comment from `# For mem-limit usages during build:` to `add optional psutil support`

Reply:
Indeed

Reply:
Well in that case for future reference, often stating the why is better than the what. The what can people read in the code themselves, the why is harder to grasp often.
Even for people who apparently wrote it themselves :rofl: